pkg/plugins/runtime/k8s/webhooks/gateway_instance_validator_test.go

```diff
@@ -100,19 +95,43 @@ var _ = Describe("MeshGatewayInstanceValidator", func() {
                   labels:
                     kuma.io/mesh: default
                 spec:
-                  selectors:
-                    - match:
-                        kuma.io/service: edge-gateway_kuma-demo_svc
-                  conf:
-                    listeners:
```

Comment:
this was just nonsense right, that's why you removed it?

Reply:
Yes, it wasn't MGI definition so I've replaced it with the real resource